Redid article scraper

AlexRavelo opened the pull request:
replaced Beautiful Soup with newspaper3k functionality

AlexRavelo committed:
Redid article scraper

replaced Beautiful Soup with newspaper3k functionality

WarmBuns commented on article-scraper.py:
```diff
@@ -1,19 +1,20 @@
+from newspaper import Article
 
 def scrapeArticle(url):
 
```
nitpick: get rid of two trailing whitespaces

WarmBuns commented on article-scraper.py:
```diff
@@ -1,19 +1,20 @@
-from bs4 import BeautifulSoup
+from newspaper import Article
 
 def scrapeArticle(url):
```
use typehints for function parameters, 
so instead of ```def scrapeArticle(url):``` 

do ```def scrapeArticle(url: str) ->  Tuple[str, str]``` make sure you import Tuple type hints at the top ```from typing import Tuple```

AlexRavelo committed:
further cleaned up article scraper

AlexRavelo committed:
Fixed article_scraper.... again

Somehow reverted article scraper to the older version last time I changed it, so I fixed it
